### ISSUE TITLE
Highlight questions in the question list

### ISSUE THREAD

**Issue by KingSupernova31** [KingSupernova31/RulesGuru#38]
When the questions list is open, have a colored highlight over the question that the user's mouse pointer is over, and also a highlight on the question that the user is currently viewing.

**Comment by bwyard** [comment]
the question list is currently a text area, you cant highlight text in text area without either  a framework or a jquery plugin. we will need to rewrite how its displayed completely, most likely using for loops and li tags if you want hover capability

**Comment by KingSupernova31** [comment]
Good point, and I think I also want it to look a bit different than I originally said anyway. I changed the description to match.
